modules/gltf/gltf_document.cpp

```diff
@@ -4213,6 +4213,35 @@ Error GLTFDocument::_parse_materials(Ref<GLTFState> p_state) {
 			}
 		}
 
+		// Supersedes KHR_materials_pbrSpecularGlossiness
+		// see https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular/README.md
```

Comment:
Do not include the trailing "/README.md", because on GitHub, linking to the folder also displays the README.

```suggestion
		// Supersedes KHR_materials_pbrSpecularGlossiness.
		// See https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular
```